<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandChainBrowserUi</class>
 <widget class="QWidget" name="CommandChainBrowserUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command Chain</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QToolButton" name="clearFilterBn">
       <property name="text">
        <string>X</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="filterCommandChain">
       <property name="styleSheet">
        <string notr="true">QComboBox {padding-left:8px;}</string>
       </property>
       <property name="maxVisibleItems">
        <number>15</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="CommandChain::LazyComboBox" name="filterTeam">
       <property name="styleSheet">
        <string notr="true">QComboBox {padding-left:8px;}</string>
       </property>
       <property name="maxVisibleItems">
        <number>15</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CommandChain::View" name="commandChainView">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="indentation">
      <number>10</number>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommandChain::View</class>
   <extends>QTreeView</extends>
   <header>View.hpp</header>
  </customwidget>
  <customwidget>
   <class>CommandChain::LazyComboBox</class>
   <extends>QComboBox</extends>
   <header>LazyComboBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
